services/ppac/src/test/java/app/coronawarn/datadonation/services/ppac/ios/verification/scenario/ratelimit/ProdPpacIosRateLimitStrategyTest.java

```diff
@@ -72,14 +63,27 @@ void shouldNotThrowExceptionWhenOnlyUpdateDayIsTheSame(int i) {
     // given
     long now = TimeUtils.getEpochSecondsForNow();
     long expirationDate = TimeUtils.getLastDayOfMonthForNow();
-    long lastUsedForPpa = LocalDateTime.now().minusDays(i).toEpochSecond(ZoneOffset.UTC);
+    long lastUsedForPpa = LocalDateTime.now(UTC).minusDays(i).toEpochSecond(UTC);
     ApiToken apiToken = new ApiToken("apiToken", expirationDate, now, null, lastUsedForPpa);
 
     // when - then
     assertThatNoException().isThrownBy(() -> {
       underTest.validateForPpa(apiToken);
     });
+  }
+
+  @Test
+  void shouldNotThrowExceptionWhenValidateForPpaIs23HoursSameDay() {
```

Comment:
The naming of the test method says it should not throw but in validateForPpa this check is made: if (hours < 23) throw. So, if hours are 23, wouldn't it throw ?

Reply:
Improved the naming of the test method. Hope this works for you? If not, feel free to suggest something else :-)